src/app/projects/components/project-settings/project-settings.component.ts

```diff
@@ -117,6 +118,9 @@ export class ProjectSettingsComponent implements OnInit, OnDestroy, AfterViewIni
   }
   labelMap: Map<string, []> = new Map<string, []>();
   @ViewChild('modalInput', { read: ElementRef }) myModalnewRecordTask: ElementRef;
+  downloadedModelsList$: any;
+  downloadedModelsQuery$: any;
+  downloadedModels: any[] = [];
```

Comment:
if this is set on every subscription this doesn't need to be initialized with []